instrumentation/rdkafka/lib/opentelemetry/instrumentation/rdkafka/patches/consumer.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module Rdkafka
+      module Patches
+        # CustomerGetter for needed for propagation
+        class CustomGetter
+          def get(carrier, key)
+            carrier[key.to_sym]
+          end
+
+          def keys(carrier)
+            carrier.keys
+          end
+        end
```

Comment:
Let's remove this and make use of the [symbol key getter](https://github.com/open-telemetry/opentelemetry-ruby/pull/1006/files#diff-680628d5c2d8484cad0bb6f9eae8dbcdb97bd635cb97c5743c438ac7fed56ee3R26-R30) added.
```suggestion
```